docs/BigQueryCreateDataset-action.md

```diff
@@ -0,0 +1,56 @@
+# BigQuery Create Dataset
+
+#### **Description**
+
+Creates a BigQuery Dataset.
+
+#### **Usage**
+
+Dataset names must be unique per project. The dataset name contain up to 1,024 characters and can
+contain letters (upper or lower case), numbers and underscore. Dataset names cannot contain special characters
+such as '-', '&', '*', or '%'.
+
+If dataset name starts with '_' (underscore) then the dataset would be hidden and not visible in
+the view and dataset names are case sensitive (E.g. 'dataset' and 'Dataset' are two different
+datasets that can co-exists in a project.
+
+#### **Basic**
+
+Following are properties used to configure this plugin
+
+* **Name**
+
+  Name of dataset to be created. May contain up to 1,024 characters and
```

Comment:
By convention, everything except the first fragment should be a complete sentence, with full punctuation, like a period at the end. 

Name of the dataset to create. The name may contain up to 1,024 characters and must consists of letters (upper or lower case), numbers, and underscores.